docs/tools/terraform/reference/cookbook/multicloud-postgresql-recipe.rst

```diff
@@ -0,0 +1,71 @@
+Deploy PostgreSQL® services to multiple clouds and regions
+==========================================================
+
+There are many reasons why businesses need to distribute their databases geographically. One of the primary reason is data residency/regulation which mandates user data to stay within certain regions. 
+This example shows a use case where a sysadmin needs to setup three highly-available PostgreSQL databases in three regions and across different cloud providers for data regulation and compliance.
```

Comment:
I'd leave "sysadmin" and either keep it neutral or address the reader directly - "This example shows how to set up". BTW, I think it is "set up", since it is a verb, not a noun here.

Reply:
This should be "set up" as @anelook says

Reply:
I see this is now updated. Resolving.